modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/form_instance_records_search_container.jsp

```diff
@@ -23,6 +23,8 @@ PortletURL portletURL = ddmFormViewFormInstanceRecordsDisplayContext.getPortletU
 	additionalProps='<%=
 		HashMapBuilder.<String, Object>put(
 			"deleteFormInstanceRecordURL", deleteFormInstanceRecordURL.toString()
+		).put(
+			"itemsTotalIds", ddmFormViewFormInstanceRecordsDisplayContext.getTotalItemsIds()
```

Comment:
@rijema please rename it to `ddmFormInstanceRecordIds`

Reply:
Sure. Thanks!